src/tinymce.js

```diff
@@ -63,8 +63,10 @@ angular.module('ui.tinymce', [])
 	        $timeout.cancel(debouncedUpdateTimer);
 	         debouncedUpdateTimer = $timeout(function() {
               return (function(ed) {
-                ed.save();
-                updateView(ed);
+                if (!ed.isNotDirty) {
```

Comment:
I think we want [isDirty()](https://www.tinymce.com/docs/api/class/tinymce.editor/#isdirty) instead.